### ISSUE TITLE
404 when using Flower 0.9.5 with --url_prefix

### ISSUE THREAD

**Issue by soer7022** [mher/flower#1001]
**Describe the bug**
When using the `--url_prefix` with Flower 0.9.5 I always get a 404 Not found error,
however when downgrading to 0.9.4 it works fine.

**To Reproduce**
1. Run Flower with --url_prefix=flower, like described in the documentation
2. Navigate to URL.com/flower
3. 404 not found

**Expected behavior**
I expect to be able to visit the flower dashboard

**System information**
Output of `python -c 'from flower.utils import bugreport; print(bugreport())'` command:
```
flower   -> flower:0.9.5 tornado:6.0.4 humanize:(0, 4)
software -> celery:4.4.5 (cliffs) kombu:4.6.10 py:3.6.9
            billiard:3.6.3.0 py-amqp:2.6.0
platform -> system:Linux arch:64bit, ELF
            kernel version:4.15.0-111-generic imp:CPython
loader   -> celery.loaders.app.AppLoader
settings -> transport:amqp results:disabled
```

**Comment by mher** [comment]
Are you using reverse proxy? Reverse proxy is not required with 0.9.5

**Comment by mher** [comment]
See https://github.com/mher/flower/pull/766 for details

**Comment by dkarchmer** [comment]
I am not using `rewrite` on my nginx, but still failing. I have

```
location /worker/ {
        proxy_pass http://flower:8888/;
        proxy_set_header Host $host;
        proxy_redirect off;
        proxy_http_version 1.1;
        proxy_set_header Upgrade $http_upgrade;
        proxy_set_header Connection "upgrade";
    }
```

and on my docker-compose

```
  flower:
    image: mher/flower:0.9.5
    environment:
      - CELERY_BROKER_URL=amqp://xxxxxxxxx
      - FLOWER_URL_PREFIX=worker
      - FLOWER_PORT=8888
    expose:
      - 8888
    depends_on:
      - rabbit1
```

and get the 404 error. How am I supposed to change this to work?

**Comment by soer7022** [comment]
I fixed it by looking at #766 

In my opinion @mher should really have mentioned this with bold letters in the change log, since it broke part of our production setup...
